extension/chrome/settings/setup/setup-import-key.ts

```diff
@@ -21,6 +22,7 @@ export class SetupImportKeyModule {
       return;
     }
     const options: SetupOptions = {
+      key_type: $('#step_2a_manual_enter .key_type').val() as KeyType,
```

Comment:
Not needed when importing, not generating a key

Reply:
Done :+:1